README.md

```diff
@@ -14,3 +14,18 @@ composer require bolt/configuration-notices
 ```
 
 ![7e22d8c8-bbdd-11e6-9668-48a1ec86ba04](https://cloud.githubusercontent.com/assets/1833361/21287029/3f4d8cf4-c463-11e6-8cd1-69583e7fa2ba.png)
+
+You can influence a few of the checks through the configuration. How meta! 
+
+```yaml
+configuration_notices:
+    local_domains: [ '.localhost' ]
+    lotsofrecords: 100
```

Comment:
I had the same thoughts.. "Limit" didn't sound right. I'll replace with `log_threshold`. 👍